### ISSUE TITLE
Building IoPay Desktop on Mac can't sent Ledger transactions

### ISSUE THREAD

**Issue by simonerom** [iotexproject/iotex-explorer#1597]
I tried building IoPay Desktop on Mac (because I needed the Ledger wallet fix #1589 but it's not released yet).

If I send a transaction from a non-ledger wallet it's working fine, also it can now login with Ledger, but when I send a transaction from the Ledger wallet I get a "Fail to broadcast action - increase gas limit or approve your Ledger and try again" error. 

Additionally, the error below is shown (unfortunately the text it's not correctly wrapped so it's not entirely readable):

![image](https://user-images.githubusercontent.com/11096047/91497098-9b49f600-e8bd-11ea-843d-5fe9649cb1eb.png)

**Comment by simonerom** [comment]
I see the new release has just been triggered a minute ago: just tried it and the error is still there.

**Comment by guo** [comment]
yep. it should work now. Feel free to close the issue if it works.

**Comment by simonerom** [comment]
Still not working, same error

**Comment by guo** [comment]
you can run iopay using command line:
```
/Applications/ioPay.app/Contents/MacOS/ioPay
```

you may see more errors here. could you please copy and paste here?

**Comment by simonerom** [comment]
When running from terminal, only startup output is shown:
```
~ » /Applications/ioPay.app/Contents/MacOS/ioPay   simone@MacBook-Pro-di-Simone
01:41:13.442 › App starting...
(electron) The default value of app.allowRendererProcessReuse is deprecated, it is currently "false".  It will change to be "true" in Electron 9.  For more information please check https://github.com/electron/electron/issues/18397
01:41:13.565 › Checking for update
service created
01:41:15.311 › Update for version 1.0.8 is not available (latest version: 1.0.8, downgrade is disallowed).
01:41:15.314 › checkForUpdatesAndNotify called, downloadPromise is null
```

After that I unlock Ledger wallet, send the transaction on IoPay, Sign on Ledger, and I get the Error in the UI but nothing gets added to the terminal output

**Comment by guo** [comment]
should have been fixed in v1.0.9
